src/index.html

```diff
@@ -5,9 +5,46 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__product">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="images-product"
+        >
+
+        <div class="card__content">
+          <h2 class="name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <h4 class="code">Product code: 195434</h4>
+
+          <div class="rating">
```

Comment:
```suggestion
          <div class="card__rating">
```

Reply:
actually you can use like you have, in BEM we can have block in block, but it's more element of card than separated block